sdk/protocol2.go

```diff
@@ -132,6 +132,19 @@ func (d *EntityData) NewMetricSet(eventType string) metric.MetricSet {
 	return ms
 }
 
+// Event method creates or retrieves an already created Event.
+func (d *EntityData) Event(summary, category string) *Event {
+	for _, e := range d.Events {
+		if e.Summary == summary && e.Category == category {
```

Comment:
I'm not sure this is the behaviour we want to adopt.

Let's say I want to log every time I restart apache. If I restart apache 5 times before flushing metrics only ` event will be reported instead of 5.

Reply:
I just realised about @areina 's comment. I wrote almost the same 🤓

Reply:
Yeah, I agree. I will remove it!